lib/spack/spack/test/packaging.py

```diff
@@ -25,7 +25,7 @@
 from spack.util.executable import ProcessError
 from spack.relocate import needs_binary_relocation, needs_text_relocation
 from spack.relocate import strings_contains_installroot
-from spack.relocate import get_patchelf, relocate_text
+from spack.relocate import get_patchelf, relocate_text, relocate_links
```

Comment:
this is minor, but when there are this many imports, I would just import `spack.relocate as rel` or something like that, and prefix the usages in the code with `rel.`

Reply:
I generally prefer to import them directly, but only for flake line-length reasons. I can double check whether we'll run into line-length problems if I change this.